docs/tutorial/part-two/index.md

```diff
@@ -407,14 +407,17 @@ Paste the following into the file:
 }
 ```
 
-Now import that file into the `about-css-modules.js` page you created earlier, by adding the following on lines 2 and 3.
-(The `console.log(styles)` code logs the resulting import so you can see what the processed file looks like).
+Now import that file into the `about-css-modules.js` page you created earlier, by editing the first few lines
+of the file to look like:
 
 ```javascript:title=src/pages/about-css-modules.js
```

Comment:
@DSchau actually I do think this is an improvement. I haven't come up with an official pattern for how to visualize changes to a file; however, in almost every other instance, we simply show the same file with the new lines highlighted.

So actually, @jimt, do you mind updating this PR to include the lines that are new in the file? Here's an example of what goes at the top of the code block
`javascript{2,5}:title=src/pages/about-css-modules.js`

Reply:
Also this corrects the error, which was that the `console.log(styles)` is not added to line 3, but rather line 5.

Reply:
Thanks for pointing out the highlighting method.